File: Server/Cinema/CinemaApp.Domain/Entities/Cinema.cs
namespace CinemaApp.Domain.Entities
{
    public class Cinema
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }
        public int CityId { get; set; }
        public City City { get; set; }
        public ICollection<Hall> Halls { get; set; }
        public ICollection<CinemaFavors> CinemaFavors { get; set; }
    }
}

File: Client/cinema-app/src/app/core/models/cinema/cinema.ts
import { CinemaFavor } from '@core/models/cinema-favor/cinema-favor';
import { City } from '@core/models/city/city';
import { Hall } from '@core/models/hall/hall';

export interface Cinema {
  id: number,
  name: string,
  address: string,
  city: City,
  halls: Hall[],
  cinemaFavors: CinemaFavor[]
}
